mmv1/third_party/terraform/tpgresource/common_diff_suppress_test.go

```diff
@@ -301,6 +301,16 @@ func TestInternalIpDiffSuppress(t *testing.T) {
 			ExpectDiffSuppress: true,
 		},
 		"same3 ipv6s": {
+			Old:                "2600:1900:4020:31cd:8000:0:0:0/96",
+			New:                "2600:1900:4020:31cd:8000::",
+			ExpectDiffSuppress: true,
+		},
+		"same4 ipv6s": {
```

Comment:
It would be nice to switch these tests to have more descriptive names now that we're getting over 3.

Reply:
done